accounts/views.py

```diff
@@ -0,0 +1,142 @@
+from rest_framework import status
+from rest_framework.authtoken.models import Token
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from .serializers import LoginSerializer
+from .models import MyUser
+from .serializers import RegistrationSerializer
+from .emailer import send_email
+
+
+class RegistrationAPIView(APIView):
+    """
+    Registers a new user.
+    """
+    serializer_class = RegistrationSerializer
+
+    def post(self, request):
+        """
+        Creates a new User object.
+        Username, email, and password are required.
+        Returns a JSON web token.
+        """
+        serializer = self.serializer_class(data=request.data)
+        serializer.is_valid(raise_exception=True)
```

Comment:
ah, actually you are already using this construction, so let's use it everywhere